examples/dnsblock/dnsblock.lua

```diff
@@ -0,0 +1,89 @@
+--
+-- SPDX-FileCopyrightText: (c) 2024 Mohammad Shehar Yaar Tausif <sheharyaar48@gmail.com>
+-- SPDX-License-Identifier: MIT OR GPL-2.0-only
+--
+
+-- Filter DNS packets based on a blocklist
+
+local xt = require("xtable")
+local linux = require("linux")
+local string = require("string")
+local action = xt.action
+local family = xt.family
+
+local udp = 0x11
+local dns = 0x35
+
+local blacklist = {
+	"github.com",
+	"gitlab.com",
+}
+
+local function nop() end
+
+local function check_blacklist (name)
+	for _,v in ipairs(blacklist) do
+		if string.find(name, v) ~= nil then
```

Comment:
are you using `find` to handle subdomains? is that the case?

Reply:
Yes